cylc/flow/task_pool.py

```diff
@@ -1755,6 +1824,22 @@ def force_trigger_tasks(
 
         return len(unmatched)
 
+    def spawn_parentless_sequential_xtriggers(self):
+        """Spawn successor(s) of parentless wall clock satisfied tasks."""
+        while self.xtrigger_mgr.sequential_spawn_next:
+            taskid = self.xtrigger_mgr.sequential_spawn_next.pop()
+            itask = (
+                self._get_hidden_task_by_id(taskid)
+                or self._get_main_task_by_id(taskid)
+            )
+            # Will spawn out to RH limit or next parentless clock trigger
+            # or non-parentless.
+            self.spawn_to_rh_limit(
+                itask.tdef,
+                itask.tdef.next_point(itask.point),
+                itask.flow_nums
+            )
+
```

Comment:
I think we might need to get `cylc remove` to pop tasks off of this list to prevent tasks from being reincarnated?